.changelog/2599.txt

```diff
@@ -0,0 +1,19 @@
+```release-note:note
```

Comment:
1.20.0-upgrade-guide.md also has to be updated.

Reply:
Added in [2c90f2e](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2599/commits/2c90f2e93886c5e464cb2d9af1c30329258125a6)